src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/GetElevationAtAPoint/GetElevationAtAPoint.xaml

```diff
@@ -0,0 +1,9 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.WPF.Samples.GetElevationAtAPoint.GetElevationAtAPoint"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:SceneView x:Name="MySceneView" />
```

Comment:
This isn't explicitly called out in the sample design, but I like to put basic instructions when the user is expected to do something. In this case I'd add something like the following:

```xml
        <Border Style="{StaticResource BorderStyle}">
            <TextBlock Text="Tap to find the elevation for a point."
                       TextAlignment="Center" TextWrapping="Wrap" FontWeight="SemiBold" />
        </Border>
```